### DISCUSSION TITLE
Removing USB Camera Support does not allow for reinstallation of USB Camera support

### DISCUSSION THREAD

**Discussion by LenBanana** [Guilouz/Creality-Helper-Script-Wiki#656] [Issues]
I would expect the script to allow for a reinstallation of the USB Camera support after I have removed it using the "Remove" option from the script, however this does not seem to be the case.

_________
Printer: Creality Ender 3 v3 Plus
Firmware: 1.2.3.8

**Comment by samschultzponsys** [comment]
Same here it just crashes the script

**Comment by paulocmorales** [comment]
Confirmed, the script crashes and takes you to the prompt

**Comment by Shavo650** [comment]
Does anyone know how to remove it manually ?

EDIT: Go to /etc/init.d/ and remove s50_usb_camera file

**Reply by LenBanana** [reply]
I ended up making a factory reset on my printer and completely reinstalled everything
